signac/contrib/project.py

```diff
@@ -2055,18 +2054,18 @@ def get_project(root=None, search=True, **kwargs):
         If no project configuration can be found.
 
     """
-    return Project.get_project(root=root, search=search, **kwargs)
+    return Project.get_project(path=path, search=search, **kwargs)
 
 
-def get_job(root=None):
+def get_job(path=None):
     """Find a Job in or above the current working directory (or provided path).
 
     Parameters
     ----------
-    root : str
-        The job root directory.
-        If no root directory is given, the current working directory is
-        assumed to be within the current job workspace directory (Default value = None).
+    path : str
+        The job directory. If no path is given, the current working
+        directory is assumed to be within the current job workspace
+        directory (Default value = None).
```

Comment:
Critical distinction!



```suggestion
    path : str
        The job directory. If no path is given, the current working
        directory is assumed to be within the current project workspace
        directory (Default value = None).
```

(If you meant "job workspace directory", that is a term we are phasing out and this should instead be "job directory".)

Reply:
I think is supposed to say "job directory."
```suggestion
    path : str
        The job directory. If no path is given, the current working
        directory is assumed to be within the current job directory
        (Default value = None).
```

Reply:
I misread the text. I made a commit with a rewording.